index.bs

```diff
@@ -3978,12 +3981,14 @@ is.
 
 Note: This is how [=array iterator objects=] work.
 For interfaces that [=support indexed properties=],
-the iterator objects returned by “entries”, “keys”, “values” and {{@@iterator}} are
+the iterator objects returned by <code class="idl">entries</code>,
+<code class="idl">keys</code>, <code class="idl">values</code>, and {{@@iterator}} are
 actual [=array iterator objects=].
 
 Interfaces with iterable declarations must not
 have any [=interface members=]
-named “entries”, “forEach”, “keys” or “values”,
+named "<code class="idl">entries</code>", "<code class="idl">forEach</code>",
```

Comment:
Is this `"<code class="idl">entries</code>"` on purpose or should it be: `"<code>entries</code>"`?

Reply:
It’s that way on purpose. I decided to use idl class for identifiers that are not plain strings.

Reply:
Mmm. You're consciously making the decision to mark those up differently than idl props (which aren't quoted), but you don't want to mark those as "infra" strings. Why not just `"entries"`, then?